perfdash/config.go

```diff
@@ -426,7 +426,8 @@ func getProwConfig(configPaths []string) (Jobs, error) {
 				continue
 			}
 			if thisPeriodicConfig.Prefix == "" || thisPeriodicConfig.Descriptions == nil {
-				return nil, fmt.Errorf("invalid perfdash config of periodic %q: none or both of prefix and job type must be specified", periodic.Name)
+				fmt.Fprintf(os.Stderr, "warning: skipping periodic %q due to invalid config\n", periodic.Name)
```

Comment:
I think we need to be more verbose here. "invalid config" can mean a lot of different things.

Please add info that "none or both of prefix and job type..." from the previous error message.